src/otaclient/boot_control/_rpi_boot.py

```diff
@@ -410,11 +415,15 @@ def prepare_tryboot_txt(self):
             logger.error(_err_msg)
             raise _RPIBootControllerError(_err_msg) from e
 
-    def reboot_tryboot(self):
+    def reboot_tryboot(self, *, reboot_wait: int = 6):
         """Reboot with tryboot flag."""
-        logger.info(f"tryboot reboot to standby slot({self.standby_slot})...")
+        logger.info(
+            f"tryboot reboot to standby slot({self.standby_slot}) in {reboot_wait=}s ..."
+        )
+        time.sleep(reboot_wait)
```

Comment:
(nits) I wonder, if we can use flush instead of waiting for constant time like the following code.
```
for handler in logger.handlers:
    handler.flush()
```

Reply:
Emmm currently our `_LogTeeHandler` at `otaclient._logging` in otaclient `v3.8.x` doesn't implement the `flush` API.

Reply:
So for v3.8.x, let's make the thing easier, just wait for 6 seconds.

And in the main branch, let's implement the `flush` API in the otaclient `LogHandler` :+1:

Reply:
Yes, this is nits comment. Please feel free to ignore as necessary. And I agree with waiting for constant value for this hot-fix.

> Emmm currently our _LogTeeHandler at otaclient._logging in otaclient v3.8.x doesn't implement the flush API.

Oh..., I understood we need to implement it by ourselves. the original `logging.Handler.flush()` just pass the function...